examples/experimental/dagster-airlift/dagster_airlift/core/airflow_cacheable_assets_def.py

```diff
@@ -396,6 +396,8 @@ def construct_assets_with_task_migration_info_applied(
                 asset.is_executable,
                 f"For an asset to be marked as migrated, it must also be executable in dagster. Found unexecutable assets for task ID {task_id}.",
             )
+            # This is suspect. Should we not be using a full AssetsDefinition copy here.
```

Comment:
worth digging into - but we would have been hitting this previously, right? so can't be where the resource was getting discarded